# HG changeset patch
# User Rik <rik@octave.org>
# Date 1499986655 25200
#      Thu Jul 13 15:57:35 2017 -0700
# Node ID 58cb2f52ce6702a9a3cdd7ae1e4d4144b98fcdf4
# Parent  528b124e337f41848a6ced61e706dffa4fbfba3a
# Parent  efba56b2406f050b0cd8b8040b27e76e7d904ee8
maint: Periodic merge of stable to default.

diff --git a/scripts/statistics/base/qqplot.m b/scripts/statistics/base/qqplot.m
--- a/scripts/statistics/base/qqplot.m
+++ b/scripts/statistics/base/qqplot.m
@@ -68,17 +68,17 @@ function [qout, sout] = qqplot (x, dist,
   endif
 
   if (nargin == 1)
     f = @stdnormal_inv;
   else
     if (isnumeric (dist))
       f = @(y) empirical_inv (y, dist);
     elseif (ischar (dist) && (exist (invname = [dist "inv"])
-                              || exist (invname = [dist "%s_inv"])))
+                              || exist (invname = [dist "_inv"])))
       f = str2func (invname);
     else
       error ("qqplot: no inverse CDF found for distribution DIST");
     endif
   endif;
 
   s = sort (x);
   n = length (x);
